eva-accession-release/src/main/java/uk/ac/ebi/eva/accession/release/io/ContigWriter.java

```diff
@@ -69,6 +71,30 @@ public void update(ExecutionContext executionContext) throws ItemStreamException
     @Override
     public void close() throws ItemStreamException {
         printWriter.close();
+        sortContigFile();
+    }
+
+    private void sortContigFile() {
+        try {
+            Process process = Runtime.getRuntime().exec(new String[]{"sort", this.output.getAbsolutePath()});
```

Comment:
instead of reading sort's standard output, why not using 
```suggestion
            String[] command = {"sort", "-o", this.output.getAbsolutePath(), this.output.getAbsolutePath()};
            Process process = Runtime.getRuntime().exec(command);
```
?
it was my bad to use `sort` without arguments but this is what I had in mind from the beginning.